integration-tests/tests/api/organization/transfer.spec.ts

```diff
@@ -342,20 +342,18 @@ test.concurrent(
 
     // current owner
     const owner = orgMembers.find(m => m.user.email === ownerEmail)!;
-    expect(orgMembers.find(m => m.id === owner.id)).toEqual(
+    expect(orgMembers.find(m => m.id === owner.id)?.role).toEqual(
       expect.objectContaining({
-        organizationAccessScopes: owner.organizationAccessScopes,
-        projectAccessScopes: owner.projectAccessScopes,
-        targetAccessScopes: owner.targetAccessScopes,
+        id: owner.role.id,
+        permissions: owner.role.permissions,
       }),
     );
```

Comment:
the simplest change here was to just replace the scope equality check with a role equality check.